packages/client/hmi-client/src/components/widgets/dropdown-button.vue

```diff
@@ -43,130 +43,117 @@ import IconCheckmark16 from '@carbon/icons-vue/es/checkmark/24';
 export type DropdownItem = {
 	displayName: string;
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	value: any;
+	value: string;
```

Comment:
The eslint-disable comment can be removed.

Reply:
done